spec/features/user_creates_a_review_spec.rb

```diff
@@ -40,4 +51,26 @@
 
     expect(page).to have_content("Rating must be between 1-5")
   end
+
+  scenario "User cannot create more than one review" do
+    expect(page).to have_field "Rating:"
+    select "3", from: "Rating:"
+    fill_in "Your Review:", with: "Man, I love this furniture so much! " \
+      "I drink so much coffee here, my kidneys are shot! Ow!"
+    click_button "Submit"
+
+    expect(page).to have_content "3"
```

Comment:
This isn't really a great assertion. The number 3 is likely to appear in the text of a page regardless of the previous interaction of the form.